spring-boot-project/spring-boot-actuator-autoconfigure/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
 	optional("io.micrometer:micrometer-core")
 	optional("io.micrometer:micrometer-tracing")
 	optional("io.micrometer:micrometer-tracing-bridge-brave")
+	optional("io.zipkin.aws:brave-propagation-aws:0.23.4") // TODO: How should we align to this version?
```

Comment:
We've got something support for keeping versions aligned in `spring-boot-dependencies`. Please add dependency management for all of the modules published from https://github.com/openzipkin/zipkin-aws to `spring-boot-dependencies`, remove the version declaration here, and we can take it from there.